src/Components/Publishing/Header/Layouts/Components/FeatureInnerContent.tsx

```diff
@@ -1,30 +1,32 @@
-import { Sans, space } from "@artsy/palette"
+import { color, Sans, space } from "@artsy/palette"
 import { unica } from "Assets/Fonts"
+import { pMedia } from "Components/Helpers"
 import { Byline } from "Components/Publishing/Byline/Byline"
 import {
   Vertical,
   VerticalOrSeriesTitle,
 } from "Components/Publishing/Sections/VerticalOrSeriesTitle"
 import React from "react"
 import styled from "styled-components"
-import { pMedia } from "../../../../Helpers"
 import { FeatureHeaderProps } from "../FeatureHeader"
 
 export const FeatureInnerContent: React.SFC<FeatureHeaderProps> = props => {
-  const { article, editTitle, editVertical } = props
+  const { article, textColor, editTitle, editVertical } = props
   const { title, hero_section } = article
-  const vertical = (article.vertical && article.vertical.name) || editVertical
+  const vertical = article.vertical && article.vertical.name
   const isFullscreen = hero_section && hero_section.type === "fullscreen"
+  const TextColor = textColor ? textColor : isFullscreen && "white"
+  const verticalColor = !vertical ? color("black30") : TextColor || undefined
 
   return (
     <TextContainer>
       <div>
         <VerticalOrSeriesTitle
           article={article}
-          color={isFullscreen ? "white" : undefined}
-          vertical={vertical}
+          color={verticalColor}
+          vertical={vertical || editVertical}
         />
-        <Title>{editTitle || title}</Title>
+        <Title color={TextColor || undefined}>{editTitle || title}</Title>
```

Comment:
why `|| undefined`?

Reply:
I get a typeerror with only TextColor than instructs me to add undefined